### PR TITLE
thwo/initial-setup

### PR DESCRIPTION
**Pull request by wogelius**
The following has been implemented:
- Composer
- Request Insurance
- Being able to trigger an incident
- Unit tests

### CODE REVIEW

File: src/PlayerAccount/PlayerAccount.php [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+use Illuminate\Http\Client\Response;
+use Illuminate\Support\Facades\Http;
+use Cego\PlayerAccount\Enumerations\Endpoints;
+use Cego\RequestInsurance\Models\RequestInsurance;
+use Illuminate\Contracts\Container\BindingResolutionException;
+use Cego\PlayerAccount\Exceptions\PlayerAccountRequestFailedException;
+
+class PlayerAccount
+{
+    protected array $headers = [
```
Comment (nbj) [inline src/PlayerAccount/PlayerAccount.php:15]: missing phpdoc

File: src/PlayerAccount/PlayerAccount.php [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+        'Content-type'  => 'application/json',
+        'Accept'        => 'application/json',
+        'Remote-User'   => 'player-account-client-dev',
+    ];
+
+    protected string $baseUrl;
```
Comment (nbj) [inline src/PlayerAccount/PlayerAccount.php:21]: missing phpdoc

File: src/PlayerAccount/PlayerAccount.php [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+    protected bool $useRequestInsurance = false;
```
Comment (nbj) [inline src/PlayerAccount/PlayerAccount.php:22]: missing phpdoc

File: src/PlayerAccount/PlayerAccount.php [hunk 4]
```diff
@@ -0,0 +1,192 @@
...
+     *
+     * @return Response|RequestInsurance
+     *
+     * @throws PlayerAccountRequestFailedException
+     */
+    protected function post(string $endpoint, array $payload)
+    {
+        if ($this->useRequestInsurance) {
+            return $this->makeRequestInsurance('post', $endpoint, $payload);
+        } else {
```
Comment (nbj) [inline src/PlayerAccount/PlayerAccount.php:122]: `else` er ikke nødvendig her

File: src/PlayerAccount/Enumerations/Incidents.php [hunk 5]
```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Cego\PlayerAccount\Enumerations;
+
+class Incidents
+{
+    // Brugeren har en spilscannerscore på over 60, har en alder på 18 - 25 og og skal kontaktes via e-mail.
```
Comment (nbj) [inline src/PlayerAccount/Enumerations/Incidents.php:7]: Alle comments skal være Engelsk, gælder hele filen

File: src/PlayerAccount/PlayerAccount.php (hunk 1 above)
Comment (wogelius) [reply src/PlayerAccount/PlayerAccount.php:25]: Fixed

File: src/PlayerAccount/PlayerAccount.php (hunk 2 above)
Comment (wogelius) [reply src/PlayerAccount/PlayerAccount.php:36]: Fixed

File: src/PlayerAccount/PlayerAccount.php (hunk 3 above)
Comment (wogelius) [reply src/PlayerAccount/PlayerAccount.php:43]: Fixed

File: src/PlayerAccount/PlayerAccount.php (hunk 4 above)
Comment (wogelius) [reply src/PlayerAccount/PlayerAccount.php]: Fixed

File: src/PlayerAccount/Enumerations/Incidents.php (hunk 5 above)
Comment (wogelius) [reply src/PlayerAccount/Enumerations/Incidents.php]: Fixed

**Pull request merged**